.github/workflows/config/mlc_config.json

```diff
@@ -2,35 +2,53 @@
     "ignorePatterns": [
         {
             "pattern": "https://github.com/Your_Github_Handle/owasp-mstg"
-        },{
+        },
+        {
             "pattern": "changelog"
-        },{
+        },
+        {
             "pattern": "https://github.com/commjoen/contributors-mstg"
-        },{
+        },
+        {
             "pattern": "http://127.0.0.1:8000"
-        },{
+        },
+        {
             "pattern": "https://haveibeenpwned.com"
-        },{
+        },
+        {
             "pattern": "https://build.frida.re/"
-        },{
+        },
+        {
             "pattern": "http://apt.thebigboss.org/repofiles/cydia/"
-        },{
+        },
+        {
             "pattern": "https://build.frida.re"
-        },{
+        },
+        {
             "pattern": "http://cydia.radare.org/debs/"
-        },{
+        },
+        {
             "pattern": "https://cydia.akemi.ai/"
-        },{
+        },
+        {
             "pattern": "https://cydia.angelxwind.net"
+        },
+        {
+            "pattern": "https://developers.android.com"
```

Comment:
If I understand properly your intention is to add the headers below so that we don't have to exclude these android domains here, right? We do want those links to be checked. Maybe some of the other options could help, have you tried?

```
  "retryOn429": true,
  "retryCount": 5,
  "fallbackRetryDelay": "30s",
  "aliveStatusCodes": [200, 206]
```

Reply:
So those headers work for 429, not for status 0. Status 0 should be ignored. I have no idea why it's not ignoring the added patterns. Maybe the PR action is being based on the owasp/mstg config.